README.md

```diff
@@ -556,21 +614,24 @@ To upgrade to version 8, the following changes should be made.
 - A `sizes` prop should be added to all usages of Imgix. If `sizes` is new to you (or even if it's not), Eric's [seminal article on `srcset` and `sizes`](https://ericportis.com/posts/2014/srcset-sizes/) is highly recommended.
 - Change all usages of `type='picture'` to `<Picture>` and `type='source'` to `<Source>`
 
-    ```html
-    // this...
-    <Imgix type='picture'>
-        <Imgix type='source' src={src}>
-        <Imgix type='source' src={src}>
-    </Imgix>
+  <!-- prettier-ignore-start -->
+
+  ```html
+  // this...
+  <Imgix type="picture">
+    <Imgix type="source" src={src} />
+    <Imgix type="source" src={src} />
+  </Imgix>
 
-    // becomes...
-    <Picture>
-        <Source src={src}>
-        <Source src={src}>
-    </Picture>
-    ```
+  // becomes...
+  <Picture>
+    <Source src={src} />
+    <Source src={src} />
+  </Picture>
+  ```
```

Comment:
Can we back out this indentation change? The intention here is to indent this codeblock so it appears as a child of the bullet point above. While GitHub somehow auto-corrects this by inferring that it's a child of the item above when rendering this Markdown document, [our own Docs page](https://docs.imgix.com/libraries/react-imgix#7x-to-80) takes the "technically correct" approach and renders it as a top-level node unless it's properly indented.